### DISCUSSION TITLE
css/styled-components-to-stylex

### DISCUSSION THREAD

**Discussion by giscus** [howdy-mj/blog-v2#62] [Announcements]
# css/styled-components-to-stylex

Next.js에서 styled-components를 StyleX로 변경하면서 느낀점

https://www.howdy-mj.me/css/styled-components-to-stylex

**Comment by hwondev** [comment]
멋진 글이네요

그냥 곁 가지 이야기로 swc와 babel이 나와서 
https://github.com/compat-table/compat-table/issues/1678

**Comment by sensecodevalue** [comment]
> SWC 지원이 빨리 되었으면 좋겠다

https://github.com/facebook/stylex/issues/70
테스트해보지는 않았지만 최근 plugin을 설치하면 할 수 있는 방향이 있는것 같아요!
